packages/manager/apps/dedicated/client/app/ip/components/list/virtual-mac/ip-ip-virtual-mac.constant.js

```diff
@@ -0,0 +1,77 @@
+  MA:
+    'https://help.ovhcloud.com/csm/fr-dedicated-servers-proxmox-network-hg-scale?id=kb_article_view&sysparm_article=KB0043915',
+  SN:
+    'https://help.ovhcloud.com/csm/fr-dedicated-servers-proxmox-network-hg-scale?id=kb_article_view&sysparm_article=KB0043915',
+  QC:
+    'https://help.ovhcloud.com/csm/fr-ca-dedicated-servers-proxmox-network-hg-scale?id=kb_article_view&sysparm_article=KB0043925',
+  CA:
+    'https://help.ovhcloud.com/csm/en-ca-dedicated-servers-proxmox-network-hg-scale?id=kb_article_view&sysparm_article=KB0043904',
+  US:
+    'https://help.ovhcloud.com/csm/en-ie-dedicated-servers-proxmox-network-hg-scale?id=kb_article_view&sysparm_article=KB0043909',
+};
+
+export const WINDOWS_SERVER_LINKS = {
+  FR:
+    'https://help.ovhcloud.com/csm/fr-dedicated-servers-hyperv-network-hg-scale?id=kb_article_view&sysparm_article=KB0043590',
+  DE:
+    'https://help.ovhcloud.com/csm/de-dedicated-servers-hyperv-network-hg-scale?id=kb_article_view&sysparm_article=KB0043579',
+  IE:
+    'https://help.ovhcloud.com/csm/en-ie-dedicated-servers-hyperv-network-hg-scale?id=kb_article_view&sysparm_article=KB0043583',
+  PL:
+    'https://help.ovhcloud.com/csm/pl-dedicated-servers-hyperv-network-hg-scale?id=kb_article_view&sysparm_article=KB0043604',
+  ES:
+    'https://help.ovhcloud.com/csm/es-es-dedicated-servers-hyperv-network-hg-scale?id=kb_article_view&sysparm_article=KB0043584',
+  IT:
+    'https://help.ovhcloud.com/csm/it-dedicated-servers-hyperv-network-hg-scale?id=kb_article_view&sysparm_article=KB0043589',
+  PT:
+    'https://help.ovhcloud.com/csm/pt-dedicated-servers-hyperv-network-hg-scale?id=kb_article_view&sysparm_article=KB0043586',
+  NL:
+    'https://help.ovhcloud.com/csm/en-gb-dedicated-servers-hyperv-network-hg-scale?id=kb_article_view&sysparm_article=KB0043582',
+  GB:
+    'https://help.ovhcloud.com/csm/en-gb-dedicated-servers-hyperv-network-hg-scale?id=kb_article_view&sysparm_article=KB0043582',
+  MA:
+    'https://help.ovhcloud.com/csm/fr-dedicated-servers-hyperv-network-hg-scale?id=kb_article_view&sysparm_article=KB0043590',
+  SN:
+    'https://help.ovhcloud.com/csm/fr-dedicated-servers-hyperv-network-hg-scale?id=kb_article_view&sysparm_article=KB0043590',
+  TN:
+    'https://help.ovhcloud.com/csm/fr-dedicated-servers-hyperv-network-hg-scale?id=kb_article_view&sysparm_article=KB0043590',
+  CA:
+    'https://help.ovhcloud.com/csm/en-ca-dedicated-servers-hyperv-network-hg-scale?id=kb_article_view&sysparm_article=KB0043581',
+  QC:
+    'https://help.ovhcloud.com/csm/fr-ca-dedicated-servers-hyperv-network-hg-scale?id=kb_article_view&sysparm_article=KB0043588',
+  US:
+    'https://help.ovhcloud.com/csm/en-ie-dedicated-servers-hyperv-network-hg-scale?id=kb_article_view&sysparm_article=KB0043583',
+  DEFAULT:
+    'https://help.ovhcloud.com/csm/en-ie-dedicated-servers-hyperv-network-hg-scale?id=kb_article_view&sysparm_article=KB0043583',
+};
+
+export const vmacFeatureAvailablity = 'ip:create-vmac';
```

Comment:
```suggestion
export const VMAC_FEATURE_AVAILABILITY = 'ip:create-vmac';
```

Reply:
Can it be more meaningful as we are displaying this only while create error with 403. Ex: VMAC_CREATION_ERROR_403 = 'ip:vmac-creation-error' 

this might need changes on 2api side as well -> create-vmac to vmac-creation-error